jenkins/scripts/bml_integration_test.sh

```diff
@@ -0,0 +1,87 @@
+#! /usr/bin/env bash
+
+set -eu
+
+# Description:
+#   Runs the integration tests for metal3-dev-env in an executer vm
+#   Requires:
+#     - source stackrc file
+#     - openstack ci infra should already be deployed.
+#     - environment variables set:
+#       - AIRSHIP_CI_USER: Ci user for jumphost.
+#       - AIRSHIP_CI_USER_KEY: Path of the CI user private key for jumphost.
+# Usage:
+#  integration_test.sh
+#
+
+CI_DIR="$(dirname "$(readlink -f "${0}")")"
+
+# shellcheck disable=SC1090
+source "${CI_DIR}/utils.sh"
+
+REPO_ORG="${REPO_ORG:-metal3-io}"
+REPO_NAME="${REPO_NAME:-metal3-dev-env}"
+REPO_BRANCH="${REPO_BRANCH:-master}"
+UPDATED_REPO="${UPDATED_REPO:-https://github.com/${REPO_ORG}/${REPO_NAME}.git}"
+UPDATED_BRANCH="${UPDATED_BRANCH:-master}"
+CAPI_VERSION="${CAPI_VERSION:-v1alpha3}"
+CAPM3_VERSION="${CAPM3_VERSION:-v1alpha3}"
```

Comment:
```suggestion
CAPI_VERSION="${CAPI_VERSION:-v1alpha4}"
CAPM3_VERSION="${CAPM3_VERSION:-v1alpha5}"
```
since new versions of CAPM3 and CAPI are released, we can change these values